src/pages/supporters.astro

```diff
@@ -0,0 +1,54 @@
+---  
+import Layout from "../layouts/Layout.astro"; 
+import { turso } from "../components/tursoClient"; 
+
+type BlorboEntry = { 
+    id: number; 
+    name: string; 
+    fandom: string; 
+    image_url: string; 
+    supporter_id: number; 
+    external_id: string; // external_id of the user
+    email: string; 
+    subscription_status: string; 
+    subscription_tier: string; 
+}; 
+
+// Execute the query and get the result 
+const result = await turso.execute(`
+  SELECT b.id, b.name, b.fandom, b.image_url, b.supporter_id, s.external_id, s.email, s.subscription_status, s.subscription_tier
+  FROM Blorbos b
+  JOIN Supporters s ON b.supporter_id = s.id
+`);
+
+// Extract rows from the result 
+const blorboEntries: BlorboEntry[] = result.rows as unknown as BlorboEntry[];
```

Comment:
I'm not 100% sure you should cast here. In theory, you should be able to get the type directly from the query, and if this cast is right then it should match BlorboEntry. Is your VSCode or whatever editor set up so you can check the TypeScript types?

If you want to change the type, you should `map` over the entry and put them in the right order.